game-core/src/main/java/games/strategy/triplea/ai/pro/ProNonCombatMoveAi.java

```diff
@@ -1170,7 +1170,8 @@ private void moveUnitsToDefendTerritories(
         }
 
         // Check if its worth defending
```

Comment:
Might be worth updating this comment to define "worth defending"

Reply:
(for potential follow-up)

Reply:
Hmm. Not sure how to do it effectively. I think the comment would end up being several lines long and essentially explaining the if conditions. This would be just the comment for the first half of the condition: "Essentially defend the territory as long as it ends up with a negative TUV for the attacker (<0) or its less than the min result (have a bunch of units that attacked into a territory so want to reinforce it for a better result)."

Thoughts?

Reply:
We certainly have danger of an out of date comment. Extracting the conditionals to a method and using intermediate values could make it self-explanatory.

As an example to outline the idea:

```
if(worthDefending( ..)) {
 areSuccessful = true;
}

:
:
private static boolean worthDefending(...) {
  boolean netTuvSwing = ...
  boolean strategicValue = ... 
  return netTuvSwing || strategicValue
}
```

Reply:
Yeah, that's probably a better option than a very long comment. Though even doing that I think it'll be difficult to fully capture all the logic/reasoning behind it.